test/unit/lib/classes/PluginManager.test.js

```diff
@@ -1837,27 +1837,47 @@ describe('PluginManager', () => {
         localPath: serviceDir,
         modules: ['local-plugin'],
       });
-      // Had to use contructor.name because the class will be loaded via
+      // Had to use constructor.name because the class will be loaded via
       // require and the reference will not match with SynchronousPluginMock
       expect(pluginManager.plugins).to.satisfy((plugins) =>
         plugins.some((plugin) => plugin.constructor.name === 'SynchronousPluginMock')
       );
     });
 
-    it('should load plugins from custom folder outside of serviceDir', () => {
-      serviceDir = path.join(tmpDir, 'serverless-plugins-custom');
-      const localPluginDir = path.join(serviceDir, 'local-plugin');
-      installPlugin(localPluginDir, SynchronousPluginMock);
+    describe('#getLocalPluginsPathPatterns()', () => {
+      it('should return empty list without any plugins', () => {
```

Comment:
We have a policy to configure all new tests with [`runServerless`](https://github.com/serverless/test/blob/master/docs/run-serverless.md). We have already plenty of such tests configured.

I think here it'll be good to rely on [`packaging`](https://github.com/serverless/serverless/tree/master/test/fixtures/programmatic/packaging) fixture, and extend configuration with local plugins that resemble direct file and directory, and confirm that in generated artifact those files are not included

Reply:
You mean an integration test? Yes, I agree. That was my first choice, but I couldn't locate any existing tests that checked packaging that I could copy from. I'll try again.

Reply:
No, I mean unit test (in sense that we can run it offline without dependency on AWS or any other remote instance).

Ideally if test is constructed with `runServerless` util, and then contents of result artifact is confirmed, in similar way as it is done in our old packaging tests: https://github.com/serverless/serverless/blob/ce66591a688ca734eb34eff929163134d1d21ae8/test/integrationPackage/lambda-files.tests.js#L52 (they should be at some point moved to regular unit tests and refactored to rely on `runServerless`)

Reply:
Yeah,I shoulda called them "eature tests" instead.

Added! Thanks for the guidance.